test/e2e/es/stackconfigpolicy_test.go

```diff
@@ -189,7 +189,7 @@ func TestStackConfigPolicy(t *testing.T) {
 					assert.NoError(t, err)
 
 					var clusterName ClusterName
-					_, err = request(esClient, http.MethodGet, "/_health_report", nil, &clusterName)
+					_, err = request(esClient, http.MethodGet, "/_cluster/health", nil, &clusterName)
```

Comment:
To just check the cluster name, `/` could also have been an option.

Reply:
aaah I didn't know that, thanks!!. do you think switching to that will be cleaner ?

Reply:
Not really "cleaner" but in a way I think it's more appropriate. I believe `/` is the most basic endpoint that has been around since the beginning and isn't going anywhere anytime soon.

I'd have written it like that:
```diff
                        test.Step{
-                               Name: "Cluster Name should be as set in config",
+                               Name: "Cluster name should be as set in the config",
                                Test: test.Eventually(func() error {
                                        esClient, err := elasticsearch.NewElasticsearchClient(es.Elasticsearch, k)
                                        assert.NoError(t, err)

-                                       var clusterName ClusterName
-                                       _, err = request(esClient, http.MethodGet, "/", nil, &clusterName)
-                                       if err != nil {
+                                       var apiResponse ClusterInfoResponse
+                                       if _, err = request(esClient, http.MethodGet, "/", nil, &apiResponse); err != nil {
                                                return err
                                        }
-
-                                       require.Equal(t, clusterNameFromConfig, clusterName.ClusterNameFromAPI)
+                                       require.Equal(t, clusterNameFromConfig, apiResponse.ClusterName)
                                        return nil
                                }),
                        },
@@ -314,8 +312,8 @@ type ClusterSettings struct {
        } `json:"persistent"`
 }

-type ClusterName struct {
-       ClusterNameFromAPI string `json:"cluster_name"`
+type ClusterInfoResponse struct {
+       ClusterName string `json:"cluster_name"`
 }
```

Reply:
sounds good. I will make that change.

Reply:
I made the suggested changes and verified locally that it works.